registrations/tasks.py

```diff
@@ -265,6 +265,34 @@ def validate(self, registration):
             self.l.info("Registration object updated.")
             return False
 
+    def create_popi_subscriptionrequest(self, registration):
+        """
+        Creates a new subscription request for the POPI message set. This
+        message set tells the user how to access the POPI required services.
+        This sould only be sent for Clinic or CHW registrations.
+        """
+        if ('prebirth' not in registration.reg_type or
+           registration.source.authority not in ['hw_partial', 'hw_full']):
+            return "POPI Subscription request not created"
+
+        self.l.info("Fetching messageset")
+        msgset_short_name = utils.get_messageset_short_name(
+            'popi', registration.source.authority, None)
```

Comment:
Are we going to create 2 new messagesets then? One for `popi.hw_partial.1` and one for `popi.hw_full.1`?
Perhaps instead we could add a check to `get_messageset_short_name` or `get_messageset_schedule_sequence` that would map the sources to the same messagset?